mcs/class/System/System.Net.Configuration/DefaultProxySection.cs

```diff
@@ -63,6 +63,7 @@ static DefaultProxySection ()
 			properties.Add (bypassListProp);
 			properties.Add (moduleProp);
 			properties.Add (proxyProp);
+			properties.Add (useDefaultCredentialsProp);
```

Comment:
@akoeplinger Yeah I meant to say something about that as well but I forgot. Thanks!